### ISSUE TITLE
Unable to reorder plugins on MO2 right hand Plugins tab!

### ISSUE THREAD

**Issue by glimmerwell** [ModOrganizer2/modorganizer#1991]
## The problem:
Briefly describe the issue you are experiencing. Tell us what you were trying to do and what happened instead. Remember, this is _not_ a place to ask questions. For that, go to our [Discord server](https://discord.gg/ewUVAqyrQX).

After updating to MO2 v.2.5.0 I can't manually sort the plugins on MO2's right side Plugins tab.

## To Reproduce:

I go to MO2 right side plugins tab where I have many mod plugins. I try to move mod plugins up or down. I am trying to both use my mouse and also the Send To > Priority function like I have done for years. Plugins are either not moving or, if I select a few at once, maybe only one moves. Some plugins won't move past the Bashed Patch, even though it is disabled. 

## Environment:

* Mod Organizer Version that exhibits the issue:  MO2 v.2.5.0
* Last Mod Organizer Version that did not exhibit the issue (if applicable):  v.2.4.4 (the one I updated from)
* Desktop OS/version used to run Mod Organizer:  Windows 10

## Details:
If necessary, describe the problem you have been experiencing with more details and pictures.

It seems that MO2 is trying to protect me from myself by not letting me... move the plugins that need to be moved. I can't temporarily move a plugin above it's Master, but if the Master is out of sight, there is no warning that the plugin did not go where I sent it to.

Also, I have installed a few MO2 Based Plugins like:

Bethesda Plugin Manager
LOOT Warning Checker
Kezyma's Curation Club
BSA Extractor 2
NIF Preview (3D Model Viewer)

But I have been removing them and I still have the issue. 

Please help, I am stopped! There has to be a way to disable this behavior.
Thanks!

## Link to Mod Organizer logs:
Right click on the log window at the bottom of MO2, set log level to debug, restart Mo2 and reproduce the problem.
To access the log files you can press the Open folder icon over the modlist and select "Open MO2 Logs folder".

### USVFS:
Create a [GIST](https://gist.github.com) which is a paste of your latest USVFS log, and link them here.
Do _NOT_ paste your full USVFS logs here, as it will make this issue very long and hard to read!
If you are reporting a bug, _always_ include the USVFS logs!

### MO Interface:
Create a [GIST](https://gist.github.com) which is a paste of your mo_interface logs, and link them here.
Do _NOT_ paste your full mo_interface logs here, as it will make this issue very long and hard to read!
If you are reporting a bug, _always_ include the mo_interface logs!

**Comment by creempie** [comment]
Same problem. I can’t work out what’s blocking the plugins.

**Comment by kencv-git** [comment]
@creempie Make sure you don't have anything in the "Filter" line below the plugins list. Be sure they are sorted by "Priority" by clicking on the priority column header. Zero at the top is best IMO. Also, MO2 enforces some rules, like masters have to load before plugins that depend on them. More info, like what game, might help if you have more trouble.

**Comment by creempie** [comment]
Yeah, I’ve realised that because of the dependencies, it won’t let you move the plugins.
You have to manually enable or disable the ESM flag.
Is there any way to remove this restriction altogether in MO2?
game: FO4

**Comment by kencv-git** [comment]
@creempie You can remove masters from the mod's esp/esm in FO4Edit by editing the Header as you probably know. I expect the enforcement in MO2 is done by the game plugin game_fallout4.dll but I'm not a programmer. The source is available on Github if you feel up to it. It might also be done by the game engine itself in which case FO4Edit would be the way to go. If the game forces the plugins to load in a different order than you had them that change will be reflected in the plugins list when you close the game.

**Comment by creempie** [comment]
I see. Thanks for your replies.
